app/geo/gui/src/ts/twxplore/gui/geo/components/filterComponent/FilterComponent.tsx

```diff
@@ -65,7 +67,7 @@ const FilterComponentImpl: React.FunctionComponent<{featureType: string}> = ({
       case TypeOfFeatureAttribute.NUMBER: {
         const numericAttributeState = attributeState as MapNumericFeatureAttributeState;
         return (
-          <div key={attributeName}>
+          <Grid item xs={12} key={attributeName}>
```

Comment:
You're using the Grid like a Bootstrap grid.

It should be
```
<Grid container direction="column">
<Grid item>
</Grid>
<Grid item>
</Grid>
...
</Grid>
```

which lays out the items top to bottom. You don't need xs={12} if you use direction="column".